src/leo-client-app/src/components/Scheduler.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+
+const Scheduler: React.FC = () => {
+
+  // TODO: Dynamicall get satelliteId from somewhere
+  const satelliteId = "655acd63d122507055d3d2ea";
+  const [validCommands, setValidCommands] = useState([]);
+  const [currentSchedule, setCurrentSchedule] = useState<string[]>([]);
+
+  const fetchValidCommands = (satelliteId: string) => {
+    fetch(`http://localhost:3001/satellite/getSatellite?satelliteId=${satelliteId}`)
+      .then((response) => response.json())
+      .then(data => {
+        setValidCommands(data.satellite.validCommands);
+      })
+      .catch((error) => {
+        console.error("Error fetching valid commands:", error);
+      }); 
+  };
+
+  const addCommand = (command : string) => {
+    setCurrentSchedule(prevCommands => [...prevCommands, command]);
+  };
+
```

Comment:
general requirement question - do we need some validation here (eg - stacking same commands, too many commands, looping?)

Reply:
not sure where we could add validation here, the most we have is just if the commands exist or not? this is just getting all the possible commands for a satellite